src/pkg/scraper/config_provider_test.go

```diff
@@ -104,7 +104,7 @@ var _ = Describe("PromScraper", func() {
 		ps, err := scraper.NewConfigProvider([]string{configGlobs}, defaultScrapeInterval, assertableLogger).Configs()
 		Expect(err).ToNot(HaveOccurred())
 		Expect(ps).To(HaveLen(0))
-		Expect(buffer.String()).To(MatchRegexp("Prom scraper config at /tmp/[0-9]*/prom_scraper_config.yml[0-9]* does not have a valid port - skipping this config file"))
+		Expect(buffer.String()).To(MatchRegexp("Prom scraper config at /.*/[0-9]*/prom_scraper_config.yml[0-9]* does not have a valid port - skipping this config file"))
```

Comment:
Maybe we should change this assertion not to match on `[0-9]` while we are here. It's not guaranteed in `os.MkdirTemp` that its a number (though it's unlikely to change).